lib/charms/opensearch/v0/helper_security.py

```diff
@@ -117,3 +123,213 @@ def to_pkcs8(private_key: str, password: Optional[str] = None) -> str:
     finally:
         os.unlink(tmp_key.name)
         os.unlink(tmp_pkcs8_key.name)
+
+
+def store_ca(
+    alias: str, store_pwd: str, store_path: str, ca: str, keep_previous: bool = True
+) -> bool:
+    """Add new CA cert to trust store."""
+    keytool = "opensearch.keytool"
+
+    if keep_previous:
+        cmd = f"{keytool} -changealias -alias {alias} -destalias old-{alias} -keystore {store_path} -storetype PKCS12"
+        args = f"-storepass {store_pwd}"
+        try:
+            run_cmd(cmd, args)
+            logger.info(f"Current CA {alias} was renamed to old-{alias}.")
+        except OpenSearchCmdError as e:
+            # This message means there was no "ca" alias or store before, if it happens ignore
+            if not (
+                f"Alias <{alias}> does not exist" in e.out
+                or "Keystore file does not exist" in e.out
+            ):
+                return False
+
+    with tempfile.NamedTemporaryFile(mode="w+t", dir=os.path.dirname(store_path)) as ca_tmp_file:
+        ca_tmp_file.write(ca)
+        ca_tmp_file.flush()
+        try:
+            run_cmd(
+                f"""{keytool} -importcert \
+                -trustcacerts \
+                -noprompt \
+                -alias {alias} \
+                -keystore {store_path} \
+                -file {ca_tmp_file.name} \
+                -storetype PKCS12
+                """,
+                f"-storepass {store_pwd}",
+            )
+            run_cmd(f"sudo chmod +r {store_path}")
+            logger.info("New CA was added to truststore.")
+        except OpenSearchCmdError as e:
+            logging.error(f"Error storing the ca-cert: {e}")
```

Comment:
There is an already defined logger, we should use it.
```suggestion
            logger.error(f"Error storing the ca-cert: {e}")
```